sklearn/isotonic.py

```diff
@@ -189,6 +189,10 @@ class IsotonicRegression(RegressorMixin, TransformerMixin, BaseEstimator):
     increasing_ : bool
         Inferred value for ``increasing``.
 
+    See Also
+    --------
+    LinearRegression : Ordinary least squares Linear Regression.
```

Comment:
Actually, the link is broken because this class is another module. We need the full module path here:

```suggestion
    sklearn.linear_model.LinearRegression : Ordinary least squares Linear Regression.
```